ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/DC1-SVC3B.cfg

```
@@ -627,7 +627,7 @@ interface Vlan3010
    no shutdown
    mtu 1500
    vrf Tenant_A_WEB_Zone
-   ip address 10.255.251.7/31
+   ip address 172.31.11.6/31
```

It should be
```suggestion
   ip address 172.31.11.7/31
```
Otherwise it is the same as the other end